front/app/components/Form/Components/Controls/SingleSelectRadioControl.tsx

```diff
@@ -76,7 +76,7 @@ const SingleSelectRadioControl = ({
               marginTop="8px"
               buttonColor={theme.colors.tenantSecondary}
               id={`${path}-radio-${index}`}
-              name="name-temp"
+              name={path}
```

Comment:
Radio inputs of the same group need unique names. Now, all radio inputs on the same page had the same names.